webapp/public/locales/en/main.json

```diff
@@ -675,6 +677,10 @@
   "studies.exportOutputFilter": "Export filtered output",
   "studies.selectOutput": "Select an output",
   "studies.variant": "Variant",
+  "studies.tree.error.failToFetchWorkspace": "Failed to load workspaces",
+  "studies.tree.error.failToFetchFolder": "Failed to load subfolders for {{path}}",
+  "studies.tree.error.detailsInConsole": "Details logged in the console",
```

Comment:
remove the `"studies.tree.error.detailsInConsole"` key as you dont use it anymore